### ISSUE TITLE
revise simsearch output

### ISSUE THREAD

**Issue by harryhoch** [monarch-initiative/monarch-legacy#555]
simsearch output currently comes in a form that is not terribly convenient for phenogrid, requiring a repackaging of the data on the client side. a revised call would handle packaging on the server, allowing phenogrid to focus solely on display. 

If other clients depend upon current simsearch formatted output, this could be done via a different API call.

**Comment by nlwashington** [comment]
can you propose a format?

**Comment by harryhoch** [comment]
@VincentIII, can you take a look at this? We can discuss after you've had a chance to chew it over.

**Comment by jmcmurry** [comment]
@harryhoch has this been addressed to your satisfaction?

**Comment by harryhoch** [comment]
@jmcmurry , not really, although some of the details of the needs might have changed.We might need some additional output from simsearch, such as subsumer paths..

**Comment by yuanzhou** [comment]
The API change suggested in #1012 will benefit phenogrid in terms of the unmatched phenotypes.
